Shared/Store/BaseDataStore.swift

```diff
@@ -421,24 +421,14 @@ extension BaseDataStore {
     }
 
     internal func handleLock() {
-        self.userDefaults.onAutoLockTime
-                .take(1)
-                .subscribe(onNext: { autoLockSetting in
-                    switch autoLockSetting {
-                    case .Never:
-                        self.unlock()
-                    default:
-                        let date = NSDate(
-                                timeIntervalSince1970: self.userDefaults.double(
-                                        forKey: UserDefaultKey.autoLockTimerDate.rawValue))
-
-                        if date.timeIntervalSince1970 > 0 && date.timeIntervalSinceNow > 0 {
-                            self.unlock()
-                        } else {
-                            self.lock()
-                        }
-                    }
-                })
-                .disposed(by: self.disposeBag)
+        let date = NSDate(
+                timeIntervalSince1970: self.userDefaults.double(
+                        forKey: UserDefaultKey.autoLockTimerDate.rawValue))
+
+        if date.timeIntervalSince1970 > 0 && date.timeIntervalSinceNow > 0 {
```

Comment:
checks that are not absolute time make me nervous, but this seems correct ...

Reply:
this code has not changed since the app launched; I'm just removing the special case for `Never` :p

Reply:
I hold myself responsible for side effects here...